src/controllers/commands/CommandController.cpp

```diff
@@ -1195,6 +1196,87 @@ void CommandController::initialize(Settings &, Paths &paths)
             crossPlatformCopy(words.mid(1).join(" "));
             return "";
         });
+
+    this->registerCommand("/color", [](const QStringList &words, auto channel) {
+        auto user = getApp()->accounts->twitch.getCurrent();
+
+        // Avoid Helix calls without Client ID and/or OAuth Token
+        if (user->isAnon())
+        {
+            channel->addMessage(makeSystemMessage(
+                "You must be logged in to use the /color command"));
+            return "";
+        }
+
+        // Colors retreived from https://dev.twitch.tv/docs/api/reference#update-user-chat-color 2022-09-11
+        const QStringList validColors{
```

Comment:
Why do we have these values stored twice? Other one is in `src/util/Twitch.cpp`. Would make sense to only have it once if twitch ever decides to ~~break~~ change something like they like to.

Reply:
The two containers serve unique purposes - for clarity I've added comments to both and moved the `validColors` container to `src/util/Twitch.cpp`